src/FakeClient/FakeClientMain.cpp

```diff
@@ -322,11 +323,9 @@ int main(int argc, char* argv[]) {
 
   std::string file_path = absl::GetFlag(FLAGS_instrument_path);
   uint64_t file_offset = absl::GetFlag(FLAGS_instrument_offset);
-  ORBIT_FAIL_IF(
-      (file_path.empty()) != (file_offset == 0),
-      "Binary path and offset of the function to instrument need to be specified together");
```

Comment:
Why is this being removed? It's a good reminder that you forgot an argument if you only specify the other one (for uprobes, you only need these two).

Reply:
Ah yeah.